src/bloom_filter/bloom_math.rs

```diff
@@ -0,0 +1,53 @@
+use super::BloomFilter;
+
+impl BloomFilter {
```

Comment:
How does moving the maths functions here help with sharing them to other possible implementations in the future? The only way this maths can be reused is if the other bloom filter implementations keep a copy of `BloomFilter` internally. I don't think it's a good idea to do that. 

Maybe you can put the maths logic in a new struct. Then it would make sense for multiple filter implementations to consume the maths logic.